LunarVim/LunarVim
# fix(cmp): jump for leading exit nodes and jump after confirm

# Description

- At some point the ability to confirm and jump to the next part of a snippet using `<CR>` was broken, this restores that behavior
- Fixes jumping after confirm for some cases where a luasnip exit node is intentionally placed prior to the cursor, such as the `import statement` snippet for `typescript`: `import { $0 } from "$1"`
- Also fixes a few LSP warnings

## How Has This Been Tested?




- Type something that prompts cmp for a snippet
- Confirm the snippet with `<CR>`
- Type something in one of the fields that may be completed
- Either abort the completion and press `<CR>` to jump or confirm it and jump to the next snippet section with `<CR>`
